bundles/CoreBundle/Command/ThumbnailsMigrateFolderStructureCommand.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Commercial License (PCL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ *  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ *  @license    http://www.pimcore.org/license     GPLv3 and PCL
+ */
+
+namespace Pimcore\Bundle\CoreBundle\Command;
+
+use League\Flysystem\FilesystemException;
+use League\Flysystem\StorageAttributes;
+use League\Flysystem\UnableToMoveFile;
+use Pimcore\Console\AbstractCommand;
+use Pimcore\Model\Asset;
+use Pimcore\Tool\Storage;
+use Symfony\Component\Console\Helper\ProgressBar;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * @internal
+ */
+class ThumbnailsMigrateFolderStructureCommand extends AbstractCommand
+{
+    protected function configure()
+    {
+        $this
+            ->setName('pimcore:thumbnails:migrate-folder-structure')
```

Comment:
```suggestion
            ->setName('pimcore:migrate:thumbnails-folder-structure')
```
Maybe we should cluster all individual migration commands in `pimcore:migrate:` for better structure? 
If ok, please also change the FQCN to `Pimcore\Bundle\CoreBundle\Command\Migrate\ThumbnailsFolderStructureCommand` and also the docs. 😉

Reply:
Applied this and also moved `pimcore:migrate:storage` to `Pimcore\Bundle\CoreBundle\Command\Migrate` namespace.